ui/admin/app/components/form/authenticate/password/index.hbs

```diff
@@ -0,0 +1,28 @@
+<Rose::Form
+  class="full-width"
+  @onSubmit={{fn this.submit @submit (hash
+    identification=this.identification
+    password=this.password
+  )}}
+  @cancel={{false}}
```

Comment:
Shouldn't this be `@cancel={{@cancel}}`?

Reply:
There is no cancel action for the auth form.